tests/bluechi_test/service.py

```diff
@@ -38,37 +46,61 @@ class Option(enum.Enum):
 
 
 class Service():
-    """An empty service, which only contains predefined sections"""
-    def __init__(self, name: str = None) -> None:
-        self.cp = configparser.ConfigParser()
-        # Enable case sensitivity
-        self.cp.optionxform = lambda option: option
-        self.name = name
+    """A base service class, which by default contains only predefined sections"""
+    def __init__(self, name: str = SERVICE_NAME, directory: str = SERVICE_DIRECTORY) -> None:
```

Comment:
```suggestion
    def __init__(self, directory: str = SERVICE_DIRECTORY, name: str = SERVICE_NAME) -> None:
```
NIT: The reversed order feels more natural to me - but thats a personal preference.

Reply:
Done